app/controllers/steps/case/date_stamp_controller.rb

```diff
@@ -1,7 +1,9 @@
 module Steps
   module Case
     class DateStampController < Steps::CaseStepController
-      def show; end
+      def show
+        @charge = current_crime_application.case.charges.create!
```

Comment:
Might be worth adding a comment about why you're putting this here, so we can decide later on about what to do with it later.